src/Misc/Master.cpp

```diff
@@ -1265,6 +1265,32 @@ bool Master::AudioOut(float *outl, float *outr)
     if(!runOSC(outl, outr, false))
         return false;
 
+    //request wavetables (ad synth) from MW if too many were consumed
+    //large for loop, but should boil down to only few iterations in practice
+    //the reason why we do it is that ADnoteParameters have no bToU
+    //(and the callbacks do not always have an rtosc::RtData object)
+    if(bToU)
+    {
+        int64_t nTimesPerSec = 5;
```

Comment:
This feels like an important magic number. Perhaps it should be promoted to somewhere a bit more likely to be seen? e.g. a class level static constant or header define?